functions-and-operators/string-functions.md

```diff
@@ -242,6 +242,7 @@ SELECT LEFT(NULL, 3);
 ### [`LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_length)
 
 Return the length of a string in bytes.
+`LENGTH()` count a multibyte character as multi bytes while `CHAR_LENGTH()` count it as a single code point
```

Comment:
Some things in Unicode make this quite complicated, for example flags.  However to me it looks like the text here is fully correct :smile: 


```
mysql> WITH n AS (SELECT 'abc' p UNION ALL SELECT 'Café' UNION ALL SELECT '😀' UNION ALL SELECT '🇪🇺') SELECT p, CHAR_LENGTH(p), LENGTH(p), HEX(p) FROM n;
+----------+----------------+-----------+------------------+
| p        | CHAR_LENGTH(p) | LENGTH(p) | HEX(p)           |
+----------+----------------+-----------+------------------+
| abc      |              3 |         3 | 616263           |                     
| Café     |              4 |         5 | 436166C3A9       |
| 🇪🇺         |              2 |         8 | F09F87AAF09F87BA |
| 😀         |              1 |         4 | F09F9880         |
+----------+----------------+-----------+------------------+
4 rows in set (0.01 sec)
```

Reply:
Note that the linter doesn't like the manual line break:

```
[2024-01-10T07:23:47.103Z] functions-and-operators/string-functions.md: this file has manual line breaks in the following lines:
[2024-01-10T07:23:47.103Z] 
[2024-01-10T07:23:47.103Z] MANUAL LINE BREAKS: L245
```

Maybe something like this, or merge it with line 244.

```suggestion

`LENGTH()` count a multibyte character as multi bytes while `CHAR_LENGTH()` count it as a single code point
```

Reply:
thanks for the review!

I agree with you and collation is more difficult for some languages related to the it...